uploading_packages/bintray/uploading_bintray.rst

```diff
@@ -56,17 +56,16 @@ repositories in the following order of priority:
   2. `conan-transit`_
   3. Your own repository
 
-If you want to have your own repository prioritized, please remove the ``conan-transit`` and
-``conan-center`` repository, then add yours first, then the others:
-
+If you want to have your own repository prioritized, please use the --insert command line option 
+when adding it
 .. code-block:: bash
 
-    $ conan remote remove conan-center
-    $ conan remote remove conan-transit
-    $ conan remote add <your_remote <your_url>
-    $ conan remote add conan-center https://conan.bintray.com
-    $ conan remote add conan-transit https://conan-transit.bintray.com
-
+    $ conan remote add <your_remote> <your_url> --insert
+    $ conan list
```

Comment:
This is wrong, it is ``conan remote list``

Reply:
fixed!